server/src/test/scala/endpoints4s/play/server/ServerInterpreterTest.scala

```diff
@@ -70,19 +70,26 @@ class ServerInterpreterTest
       serverApi.routesFromEndpoints(endpoint.implementedByAsync(logic))
     )(runTests)
 
+  def serveManyEndpoints(endpoints: EndpointWithImplementation*)(runTests: Int => Unit): Unit = {
+    serveRoutes(
+      serverApi.routesFromEndpoints(endpoints.map(e => e.endpoint.implementedBy(e.impl)): _*)
+    )(runTests)
+  }
+
+  val port = {
+    val socket = new ServerSocket(0)
+    try socket.getLocalPort
+    finally if (socket != null) socket.close()
+  }
+  val config = ServerConfig(mode = Mode.Test, port = Some(port))
+  var routes: PartialFunction[RequestHeader, Handler] = { case _ =>
+    ???
```

Comment:
This works because the routes are re-assigned to the ones passed as a parameter on line 91, right?

Reply:
yep

Reply:
it speeds up the tests by about 50%